src/transaction_builder.js

```diff
@@ -238,7 +238,7 @@ TransactionBuilder.prototype.__build = function (allowIncomplete) {
         case 'multisig': {
           // Array.prototype.map is sparse-compatible
           var msSignatures = input.signatures.map(function (signature) {
-            return signature.toScriptSignature(input.hashType)
+            return signature && signature.toScriptSignature(input.hashType)
```

Comment:
My only qualm was that I don't think this is necessary.  However I know it is necessary given the current solution of pre-emptively always putting in the `OP_0`'s.  I might change that and submit a PR.